pkg/apis/componentconfig/validation/validation.go

```diff
@@ -93,17 +70,20 @@ func ValidateKubeSchedulerLeaderElectionConfiguration(cc *componentconfig.KubeSc
 // ValidateLeaderElectionConfiguration ensures validation of the LeaderElectionConfiguration struct
 func ValidateLeaderElectionConfiguration(cc *componentconfig.LeaderElectionConfiguration, fldPath *field.Path) field.ErrorList {
 	allErrs := field.ErrorList{}
-	if cc.LeaseDuration.Duration <= 0 {
-		allErrs = append(allErrs, field.Invalid(fldPath.Child("leaseDuration"), cc.LeaseDuration, "must be greater than or equal to 0"))
+	if !cc.LeaderElect {
+		return allErrs
+	}
+	if cc.LeaseDuration.Duration < 1 {
```

Comment:
Actually, the error condition should remain as `cc.LeaseDuration.Duration <= 0` here and below.